spec/factories/vacations.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :vacation do
+    start_date { Date.today }
+    end_date { 2.months.from_now }
+    status { 1 }
+    user { nil }
```

Comment:
```suggestion
    user
```

Since the `user` is required for a Vacation to be valid, you can just associate it in the factory, so that you won't have to do that for every `Vacation` object generated from the factory